database/index.js

```diff
@@ -0,0 +1,10 @@
+const mongoose = require('mongoose');
+mongoose.Promise = require('bluebird');
```

Comment:
bluebird being used in this file?

Reply:
Added bluebird to promisify mongoose operations using bluebird library.